site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -238,6 +238,101 @@ Apply an Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer
 curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/annotation
 ```
 
+### Counters
+
+{{< alpha title="Counters" gate="Counters" >}}
+
+**Note:** In the Counters examples, we retrieve the counter under the key "conformanceTestCounter" to make it extra clear.
+
+#### Alpha: GetCounter
+This function retrieves a specified counter by its name and returns its information.
+
+##### Example
+
+```bash
+curl -H "Content-Type: application/json" -X GET http://localhost:${AGONES_SDK_HTTP_PORT}/v1alpha1/counters/conformanceTestCounter
```

Comment:
```suggestion
curl -H "Content-Type: application/json" -X GET http://localhost:${AGONES_SDK_HTTP_PORT}/v1alpha1/counters/rooms
```

(and all the other references)

Reply:
Renamed to `rooms` in all the references